Auto Layout/CollectionViewController.swift

```diff
@@ -46,6 +33,49 @@ final class CollectionViewController: UICollectionViewController, UICollectionVi
     collectionView.delegate = self
     collectionView.register(cell: HeadlineSummaryCell.self)
     collectionView.register(cell: ThumbnailCell.self)
+    collectionView.register(cell: LargeImageCell.self)
+  }
+
+  private static var layout: UICollectionViewLayout {
```

Comment:
could these be static `let`s instead of vars? we might save some cycles that way